pages-functions-static-form/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I know the recommended `tsconfig.json` of @cloudflare/workers-types is a bit more bare-bones. Does it make sense to add any of these fields to that file? I'm copying from there what seems like once-a-day.

Reply:
Right, this is what wrangler2 generates. It would make sense to consolidate.

Reply:
adding the minimal example for now